src/fhost_fpga.py

```diff
@@ -336,6 +336,23 @@ def qdr_okay(self):
         LOGGER.info('%s: QDR okay.' % self.host)
         return True
 
+    def check_fft_overflow(self):
+        """
+        Checks if pfb counters on f-eng are not incrementing
```

Comment:
The docstring is a bit confusing on what the return value means. Does True mean they are overflowing, or that they are OK (i.e. not overflowing)?